README.md

```diff
@@ -34,9 +34,16 @@ The design philosophy is described in an [arXiv paper](https://arxiv.org/abs/180
 Glow builds and runs on macOS and Linux. The software depends on a modern C++
 compiler that supports C++11, on CMake, LLVM, protocol buffers, and libpng.
 
+#### Get Glow!
+
+  ```
+  git clone git@github.com:pytorch/glow.git  # or, if you prefer: git clone https://github.com/pytorch/glow.git
```

Comment:
nit, you can specify that it's bash. by using ``` followed by bash.

```bash
git clone git@github.com:pytorch/glow.git  # or, if you prefer: git clone https://github.com/pytorch/glow.git
cd glow
```

Maybe change the rest as well.

Reply:
I'll do that when I update this PR. Thanks Roman.